src/js/htmlutils.js

```diff
@@ -61,18 +61,20 @@ var htmlUtils = exports.htmlUtils = {
    * @param {String} origin The origin to get description for.
    * @returns {String} Localized action description with origin.
    */
-  getActionDescription: (function () {
```

Comment:
Looks like we had a bad merge here. Could you restore my changes from 7fddc74cba7d5e5d8721c4bfb2947a726d7092c2?